.changes/unreleased/Features-20231107-132842.yaml

```
@@ -0,0 +1,6 @@
+kind: Features
+body: add flag --no-skip-on-failture
```

```suggestion
body: add flag --no-skip-on-failure
```

"failture" should be changed to "failure" here and at all other usages.